cmd/root.go

```diff
@@ -136,6 +136,7 @@ func init() {
 		"Path to the template on disk")
 	rootCmd.PersistentFlags().StringP("saved-templates-path", "T", "/kubectl-build-deploy/lagoon/services-routes",
 		"Path to where the resulting templates are saved")
```

Comment:
I wonder, as a general thing - whether this needs some kind of documentation?
I understand why the default isn't set here (and is set generator.go, I believe) - but is there any signposts as to what the structure of this argument should be (which would at least be implied by a default value)

Reply:
wrong line